geonode/layers/views.py

```diff
@@ -1502,28 +1490,23 @@ def layer_thumbnail(request, layername):
         raise Http404(_("Not found"))
 
     try:
+        # try:
```

Comment:
Why are those commented parts of the code in this PR?

Reply:
Custom if-else statements used by leaflet only. Bad practice indeed. The idea is to remove all those if-else hardcoded into the code.